CPT331.Web.Tests/Controllers/HomeControllerTest.cs

```diff
@@ -19,7 +19,7 @@ public void Index()
 			HomeController controller = new HomeController();
 
 			// Act
-			ViewResult result = controller.Index() as ViewResult;
+			ViewResult result = controller.Home() as ViewResult;
```

Comment:
Change 'Index' to 'Home' - they mean different things to me, an index page is not always a home page etc.